src/Command/TwoStepVerificationCommand.php

```diff
@@ -16,17 +16,16 @@
 use FOS\UserBundle\Model\UserManagerInterface;
 use Sonata\UserBundle\GoogleAuthenticator\Helper;
 use Sonata\UserBundle\Model\UserInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
```

Comment:
As the comment says, it was supposed to be changed in the next major.
BTW, I think we must mark this command as final, using the `@final` annotation until the next major.

Reply:
any idea when the `next major` will be released? anyone started to work on that?

Reply:
Or can I do this MR to `master` ?

Reply:
Yes you can.

The next major will be released when every NEXT_MAJOR will be resolve + some others things, son any help is good to take.

We first try to release SonataAdmin 4.0

Reply:
@phansys I'm closing thing thread could you see again, my new changes?